### DISCUSSION TITLE
Add export park options

### DISCUSSION THREAD

**Discussion by That-Human-Being** [OpenRCT2/OpenRCT2#18745] [Feature requests]
I brought this up on discord, and Gymnasiast said it would be easy to hack the park exporter, while the question was about not saving the pause state, it would also be beneficial to make sure that custom objects are included irregardless of what the setting is, and automatically running `remove_unused_objects`

My Question:
> How do I have a file that will unpause after I save it, I want to export something that has a message that I want to play as soon as it is opened

Gymnasiast's response:
> Short of hacking the park file exporter (which is not difficult for this kind of thing), I don’t think there is a way.

**Comment by Richard-L** [comment]
There's already this which seems to have certain overlap https://github.com/OpenRCT2/OpenRCT2/discussions/17856

**Reply by That-Human-Being** [reply]
I would rather have a separate option to export a final version than to have to change settings before and after I save, I like the preserved pause state most of the time, it is only when I finish and want to add a welcome notification for when people open it that this becomes a problem
